hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/autoassign/AutoAssignCheckerTest.java

```diff
@@ -243,6 +243,28 @@ public void checkAutoAssignWithDifferentActionTypes() {
         verifyThatTargetsHaveAssignmentActionType(ActionType.SOFT, targetsB);
     }
 
+    @Test
+    @Description("Test auto assignment of a distribution set with DOWNLOAD_ONLY action type")
+    public void checkAutoAssignWithDownloadOnlyActionType() {
```

Comment:
I would not define additional test case here for the `DownloadOnly` action type. There is already a test case above that validates allowed action types, so I would rather extend the previous one and extract the duplicate code in a helper method.